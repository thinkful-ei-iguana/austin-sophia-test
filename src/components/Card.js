import React from 'react';
import store from '../components/store';


export default function App(props) {
    //let card = store.allCards[props.cardId];
    return (
        <div className="Card">
            <h3>{props.title}</h3>
            <p>{props.content}</p>
            <button>Delete</button>
        </div>
    );
}


