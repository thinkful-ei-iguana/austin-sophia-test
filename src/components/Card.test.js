import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card';
import store from './store'

//import { exportAllDeclaration } from '@babel/types';




it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
}); 

it('renders the UI as expected', () =>{

    const tree = renderer
      .create(<div class="Card"><h3>First card</h3><p>lorem ipsum</p><button>Delete</button></div>)
      .toJSON();
    expect(tree).toMatchSnapshot();
});

