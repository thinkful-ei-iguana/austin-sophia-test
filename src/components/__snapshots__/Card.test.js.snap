// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  class="Card"
>
  <h3>
    First card
  </h3>
  <p>
    lorem ipsum
  </p>
  <button>
    Delete
  </button>
</div>
`;
